apps/passport-server/src/database/queries/rateLimit.ts

```diff
@@ -0,0 +1,20 @@
+import { Pool } from "postgres-pool";
+import { RateLimit } from "../models";
+import { sqlQuery } from "../sqlQuery";
+
+export async function checkRateLimit(
+  client: Pool,
+  eventType: RateLimit["type"],
+  eventId: string
```

Comment:
thought: depending on whether you choose to use the `VARCHAR(100)` type or not, it could make sense to enforce a length limit here too.